website/docs/r/cluster.html.markdown

```diff
@@ -10,18 +10,26 @@ description: |-
 
 Provides RKE cluster resource. This can be used to create RKE clusters and retrieve their information.
 
+RKE clusters can be configured in distinct ways:
+- Using `cluster_yaml` argument: On this mode, can use a full RKE cluster config yaml file.
```

Comment:
Using `cluster_yaml`: The full RKE cluster is defined in an RKE cluster.yml file.
Using the TF provider arguments to define the entire cluster.
Using a combination of both the `cluster_yaml` and TF provider arguments. The TF arguments will override the `cluster_yaml` options if collisions occur.

Reply:
Updated